src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -576,8 +576,6 @@ public IActionResult GetHistory([FromQuery] WalletHistoryRequest request)
                         }
                     }
 
-                    transactionItems = transactionItems.Distinct(new SentTransactionItemModelComparer()).Select(e => e).ToList();
```

Comment:
One 'sent' transaction can consume multiple inputs from the wallet. These inputs would all have identical spending details, and therefore should be deduplicated from the overall list.

Reply:
Updated logic to take index into consideration